zerver/models.py

```diff
@@ -931,6 +931,54 @@ def flush_per_request_caches() -> None:
     per_request_realm_filters_cache = {}
 
 
+class RealmPlayground(models.Model):
+    """Server side storage model to store playground information needed by our
+    'view code in playground' feature in code blocks.
+    """
+
+    MAX_PLAYGROUND_NAME_LENGTH = 60
+    MAX_PYGMENTS_LANGUAGE_LENGTH = 40
+
+    realm: Realm = models.ForeignKey(Realm, on_delete=CASCADE)
+    url_prefix: str = models.TextField(validators=[URLValidator()])
+
+    # User-visible display name used when configuring playgrounds in the settings page and
+    # displaying them in the playground links popover.
+    name: str = models.CharField(db_index=True, max_length=MAX_PLAYGROUND_NAME_LENGTH)
```

Comment:
This mostly doesn't matter since this table will be tiny, but we probably want to use `TextField` for this -- in that most names will be much shorter than the max length, and so it's more efficient space-wise (and if we change our mind about the length) to use that structure instead.